The buildbot/AMD64 Windows10 PR failure is an unrelated memory error:
```
======================================================================
ERROR: test_stack_overflow (test.test_compile.TestSpecifics)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "D:\buildarea\pull_request.bolen-windows10\build\lib\test\test_compile.py", line 718, in test_stack_overflow
    compile("if a: b\n" * 200000, "<dummy>", "exec")
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
MemoryError
```

--- a/Lib/test/support/__init__.py
+++ b/Lib/test/support/__init__.py
@@ -2070,13 +2070,13 @@ def clear_ignored_deprecations(*tokens: object) -> None:
         raise ValueError("Provide token or tokens returned by ignore_deprecations_from")
 
     new_filters = []
+    endswith = tuple(rf"(?#support{id(token)})" for token in tokens)
     for action, message, category, module, lineno in warnings.filters:
         if action == "ignore" and category is DeprecationWarning:
             if isinstance(message, re.Pattern):
                 msg = message.pattern
             else:
                 msg = message or ""
-            endswith = tuple(rf"(?#support{id(token)})" for token in tokens)
             if msg.endswith(endswith):
                 continue
         new_filters.append((action, message, category, module, lineno))

